.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+jobs:
+    build:
+        docker:
+            - image: circleci/node:8
+        steps:
+            - checkout
+            - restore_cache:
+                key: dependency-cache-{{ checksum "package.json" }}
+            - run: npm install
+            - save_cache:
+                key: dependency-cache-{{ checksum "package.json" }}
+                paths:
+                    - ./node_modules
+            - run: npm test
+            - run: npm run semantic-release
```

Comment:
We definitely do not want this in the `build` job.  Please check out an existing release config --> https://github.com/electron/asar/blob/master/.circleci/config.yml#L25-L64